server/plugins/builder/pages/summary.js

```diff
@@ -81,20 +92,23 @@ class SummaryPage extends Page {
       return h.view('summary', viewModel)
     }
   }
-
-  // get stateSchema () {
-  //   const keys = this.compontStateSchemaKeys()
-  //   const name = this.components.formItems[0].name
-  //   const d = new Date()
-  //   d.setDate(d.getDate() + 28)
-  //   const max = `${d.getMonth() + 1}-${d.getDate()}-${d.getFullYear()}`
-
-  //   // Extend the key to validate that the date is
-  //   // greater than today and less than today+28 days
-  //   keys[name] = keys[name].min('now').max(max)
-
-  //   return joi.object().keys(keys)
-  // }
+  makePostRouteHandler (getState) {
+    return async (request, h) => {
+      const model = this.model
+      model.basePath = h.realm.pluginOptions.basePath || ''
+      const state = await model.getState(request)
+      const { applicableFees } = new SummaryViewModel(model, state)
+      const reference = `FCO-${shortid.generate()}`
```

Comment:
What if no applicable fees?

Reply:
what do you mean? as in we should tell the person building the form that no fee is presented? (nothing) 

or what happens in the view? applicableFees is initialised as an empty array and summary.html checks if the array has a length and presents a button

Reply:
I mean if this endpoint was posted to but the form has no fees configured it'll cause an error. Maybe not a problem as if it's posted to with no fees it means someone has hacked the form, but we should probably handle it nicely rather than error.